src/core/annotation.js

```diff
@@ -439,13 +439,22 @@ class Annotation {
     );
   }
 
-  isHidden(annotationStorage) {
+  isPrintable(annotationStorage) {
+    const storageEntry =
+      annotationStorage && annotationStorage.get(this.data.id);
+    if (storageEntry && storageEntry.print !== undefined) {
+      return storageEntry.print;
+    }
+    return this.printable;
+  }
+
+  isRenderable(annotationStorage) {
```

Comment:
Nit: `isViewable` instead, such that it matches the `viewable` property used below?